packages/frontend/core/src/components/affine/reference-link/index.tsx

```diff
@@ -128,21 +119,11 @@ export function AffinePageReference({
 
   const ref = useRef<HTMLAnchorElement>(null);
 
+  const [refreshKey, setRefreshKey] = useState<string>(() => nanoid());
```

Comment:
make a standalone hook?
```
const refresh = useForceRefresh()
```

Reply:
<div class='graphite__hidden'>
          <div>🎥 Video uploaded on Graphite:</div>
            <a href="https://app.graphite.dev/media/video/g3jz87HxbjOJpXV3FPT7/503335bd-0766-48c9-bae5-4a1108dab272.mp4">
              <img src="https://app.graphite.dev/api/v1/graphite/video/thumbnail/g3jz87HxbjOJpXV3FPT7/503335bd-0766-48c9-bae5-4a1108dab272.mp4">
            </a>
          </div>
<video src="https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/g3jz87HxbjOJpXV3FPT7/503335bd-0766-48c9-bae5-4a1108dab272.mp4">CleanShot 2024-09-04 at 13.37.52.mp4</video>

This is to add a key to the url so that each click can trigger the focus action even if the blockid does not change.